cmd/evm/runner.go

```diff
@@ -274,14 +275,18 @@ func runCmd(ctx *cli.Context) error {
 	if ctx.Bool(CreateFlag.Name) {
 		input = append(code, input...)
 		execFunc = func() ([]byte, uint64, error) {
+			// don't mutate the state!
+			runtimeConfig.State = prestate.Copy()
```

Comment:
@jwasinger @holiman 

Why was this done here?

This makes it impossible to re-use the state dump. The account still gets inserted however instead of an address I now see:

`"pre(0xdfeeda06f79105a5aa975e5ef88e1b24d0062f31b5f42850e35d9cd4596df77e)": {`

Which means I can no longer take the state dump and execute another call based on the state dump.

Reply:
Could you please open an issue with this, so we wont forget about it

Reply:
@jwasinger thanks for pointing this out, got it. #31151 fixes this by only copying it during benchmarks.

Reply:
@MariusVanDerWijden since the fix looks obvious to me I decided to open a small PR instead (#31151).